content/appeals/decision-reviews/release-notes/2025-08-08.md

```diff
@@ -0,0 +1,17 @@
+The Final Status (final_status) attribute has been added to the following evidence submission endpoints of the Decision Reviews API:
```

Comment:
It was called `final_status` in Benefits Intake, but the fields in Decision Reviews are camel-case rather than snake-case by default, so it should be `finalStatus` here instead.